File: src/main/java/net/hectus/neobb/turn/person_game/categorization/SituationalAttackCategory.java
package net.hectus.neobb.turn.person_game.categorization;

import org.bukkit.Material;

public interface SituationalAttackCategory extends AttackCategory {
    @Override
    default int categoryMaxPerDeck() {
        return 1;
    }

    @Override
    default Material categoryItem() {
        return Material.SNOWBALL;
    }

    @Override
    default String categoryName() {
        return "situational_attack";
    }
}

File: src/main/kotlin/net/hectus/neobb/modes/turn/person_game/categorization/SituationalAttackCategory.kt
package net.hectus.neobb.modes.turn.person_game.categorization

import org.bukkit.Material

interface SituationalAttackCategory : AttackCategory {
    override val categoryMaxPerDeck: Int get() = 1

    override val categoryItem: Material get() = Material.SNOWBALL
    override val categoryName: String get() = "situational_attack"
}
